contingency/contingency-api/src/main/java/com/powsybl/contingency/ContingencyList.java

```diff
@@ -72,4 +74,12 @@ static ContingencyList load(String filename, InputStream stream) {
     static ContingencyList of(Contingency... contingencies) {
         return new DefaultContingencyList(contingencies);
     }
+
+    static List<Contingency> checkValidity(List<Contingency> contingencies, Network network) {
```

Comment:
Make this method public so I could use it. I thought we agreed that the name was not well chosen